common/changes/@microsoft/gulp-core-build/ts-snap_2017-12-28-10-53.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/gulp-core-build",
+      "comment": "Respect the snapshot base file name.",
```

Comment:
>Respect the snapshot base file name. [](start = 18, length = 36)

"Fix an issue where snapshot filenames did not follow Jest's naming convention."